portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,24 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&family=IBM+Plex+Sans&display=swap');
+
+:root, body, html {
+  font-size: 16px;
+  font-family: 'IBM Plex Sans', sans-serif;
+}
+
+header, footer {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  max-width: 100vh;
+  width: 80rem;
+  padding: 0 2rem;
```

Comment:
Any reason you're using `rem`'s? It'll be helpful to scale with the root font size if you expect it to be variable, although you may find it easier to scale things with `px` since that will remain constant between projects. Not required, but a suggestion since I noticed your later PR's use `rem`'s exclusively.

Reply:
I'm in the habit of using `rem`s as much as possible just because it makes it easier to scale things for accessibility / tweaking the UI / responsiveness. I always set the root font size to 16px anyway, so I don't really think consistency will be an issue, but thanks for the tip.